src/components/DataSection/DataSection.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import './DataSection.scss';
+import { getLocation } from '../../commons/helpers';
+import Card from '../Card/Card';
+import { Tabs } from '../../statics/Statics';
+
+function DataSection ({activeTab}) {
+    const [data, setData] = useState([]);
+    const [loading, setLoading] = useState(false);
+
+    useEffect(() => getData(), [activeTab])
+
+    const callback = (results, status) => {
+     if (status === window.google.maps.places.PlacesServiceStatus.OK) {
+            setData(results)
+            setLoading(false)
+        }
```

Comment:
Please implement error handling here. It the status is not OK, the user will end up staring at the loader.

Reply:
Done.